monitoring/bin/remove_monitoring_tenant.sh

```diff
@@ -12,6 +12,14 @@ cd "$(dirname $BASH_SOURCE)/../.."
 source monitoring/bin/common.sh
 source bin/openshift-include.sh
 
+if [ "$OPENSHIFT_CLUSTER" == "true" ]; then  
+  if [ "${CHECK_OPENSHIFT_CLUSTER:-true}" == "true" ]; then
+    log_error "This script should not be run on OpenShift clusters"
```

Comment:
Probably should be reworded to make it more clear that it appears (or we believe) that the current environment is OpenShift.